fpdf/table.py

```diff
@@ -819,6 +840,7 @@ def cell(
             rowspan,
             padding,
             link,
+            border,
```

Comment:
```suggestion
            CellBordersLayout.coerce(border),
```
It's better to check if the value provided is valid early, and to always store an instance of `CellBordersLayout`.